admin-dashboard/docs/plan/dependency-graph-redesign.md

```
@@ -0,0 +1,167 @@
+- [ ] Iconos son **símbolos SVG técnicos** (no emojis)
+- [ ] Layout es **horizontal** (left → right flow)
+- [ ] Conexiones tienen **flechas claras** con curvas suaves
+- [ ] Estilo visual coincide **90%+** con la imagen de referencia
+- [ ] Hover states son **suaves y profesionales**
+- [ ] Click en un nodo **resalta todo su path** (dependencies + dependents)
+- [ ] El grafo es **responsive** (adapta a mobile)
+- [ ] Performance es **óptima** (no lag con 13 nodos)
+
+## 🔄 Workflow de Ejecución
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)